common.js

```diff
@@ -0,0 +1,48 @@
+import axios from 'axios';
+
+const METRICS_URL = process.env.METRICS_URL;
+const METRICS_AUTH_TOKEN = process.env.METRICS_AUTH_TOKEN;
+const METRICS_TAG = process.env.METRICS_TAG;
+
+// Victoria metrics helper functions
+function getPrometheusMetricPushPayload(name, labels, value) {
+    const labelsStrs = [];
+    
+    for (const [k, v] of Object.entries(labels)) {
+        if (!v) {
+            continue;
+        }
+        labelsStrs.push(`${k}="${v}"`);
+    }
+    
+    const allLabelsStr = labelsStrs.join(",");
+    return `${name}{${allLabelsStr}} ${value}`;
+  }
+  
+  async function pushPrometheusMetricsToVM(payloads) {
```

Comment:
nit: rename this to something more generic? like `pushPrometheusMetrics`

Reply:
done